### PR TITLE
Update theme

### PR DESCRIPTION
**Pull request by umarcor**
This PR is based on some of the commits from #521.

The theme is updated to use the version from https://github.com/SymbiFlow/sphinx_symbiflow_theme instead of http://github.com/SymbiFlow/sphinx_materialdesign_theme (which was renamed to https://github.com/SymbiFlow/sphinx_symbiflow_theme-old).

The result is similar to [symbiflow-examples.rtfd.io](https://symbiflow-examples.rtfd.io). See https://symbiflow--522.org.readthedocs.build/en/522/

Ref:

- SymbiFlow/sphinx_symbiflow_theme#7

### CODE REVIEW

File: .github/workflows/Doc.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+name: Docs
+
+on:
+  pull_request:
+  push:
+  schedule:
+    - cron: '0 0 * * 4'
```
Comment (kgugala) [inline .github/workflows/Doc.yml:7]: maybe we should rebuild the docs daily?

**Review by kgugala — APPROVED** [review]
LGTM, one small comment about build schedule

File: .github/workflows/Doc.yml (hunk 1 above)
Comment (umarcor) [reply .github/workflows/Doc.yml:7]: @kugala, note that at the moment this build is *not* the upstream. I added it in order for contributors to have CI on their forks without registering a new project in RTD and without being forced to open PRs. However, the "official" build is done externally, on RTD's servers/service. As far as I am aware, RTD does not support pushing externally built artifacts.

Therefore, I will change it to a daily cron, but it is going to be relevant for continuous testing only, not for continuous deployment.

Comment (umarcor) [reply .github/workflows/Doc.yml]: Done in https://github.com/SymbiFlow/symbiflow-docs/pull/522/commits/a51155bf4c591c106bd2b91cf8ba3103a652523d.

**Pull request merged**